### PR TITLE
Add zone less events directive

### PR DESCRIPTION
**Pull request by BioPhoton**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,8 +2,7 @@
 
 [![rx-angular](https://circleci.com/gh/BioPhoton/rx-angular.svg?style=shield)](https://circleci.com/gh/BioPhoton/rx-angular)
 
-
-This repository holds a set of helpers to create fully reactive applications.
+This repository holds a set of helpers to create **fully reactive** as wel as **fully zone-less** applications.
```
Comment (BioPhoton) [inline README.md:5]:
```suggestion
This repository holds a set of helpers to create **fully reactive** as well as **fully zone-less** applications.
```

File: README.md [hunk 2]
```diff
@@ -3,7 +3,8 @@
 [![rx-angular](https://circleci.com/gh/BioPhoton/rx-angular.svg?style=shield)](https://circleci.com/gh/BioPhoton/rx-angular)
 
 
-This repository holds a set of helpers to create fully reactive applications.
+This repository holds a set of helpers to create **fully reactive** as wel as  **fully zone-less** applications.
```
Comment (hoebbelsB) [inline README.md]:
```suggestion
This repository holds a set of helpers to create **fully reactive** as well as  **fully zone-less** applications.
```

didn't we talk about those bold stuff? :D

File: libs/template/src/lib/core/utils/make-zone-less.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
+import { apiZonePatched, getGlobalThis } from '@rx-angular/template';
```
Comment (hoebbelsB) [inline libs/template/src/lib/core/utils/make-zone-less.ts:1]: i am pretty sure those imports should be relative

File: libs/template/src/lib/core/utils/make-zone-less.ts [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+ * @description
+ *
+ * This function takes an elem and event and re-applies the listeners from the passed event to the
+ * passed element with the zone un-patched version of it.
+ *
+ * @param elem {HTMLBaseElement} - The elem to re-apply the listeners to.
+ * @param event {string} - The name of the event from which to re-apply the listeners.
+ *
+ */
+export function reapplyEventListenerZoneUnPatched(
```
Comment (hoebbelsB) [inline libs/template/src/lib/core/utils/make-zone-less.ts:33]: what about `revertZonePatch` or `unDoZonePatch` ?

File: libs/template/src/lib/core/utils/make-zone-less.ts [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+  elem: HTMLBaseElement,
```
Comment (hoebbelsB) [inline libs/template/src/lib/core/utils/make-zone-less.ts:34]: i am pretty sure u want `HTMLElement`, not HTMLBaseElement

File: README.md (hunk 2 above)
Comment (BioPhoton) [reply README.md]: Ture :D lets fix it in a release branch.

File: libs/template/src/lib/core/utils/make-zone-less.ts (hunk 4 above)
Comment (BioPhoton) [reply libs/template/src/lib/core/utils/make-zone-less.ts]: renamed to revertEventListenerZonePatch

File: libs/template/src/lib/core/utils/make-zone-less.ts (hunk 5 above)
Comment (BioPhoton) [reply libs/template/src/lib/core/utils/make-zone-less.ts]: Yes!

File: libs/template/src/lib/core/utils/make-zone-less.ts (hunk 3 above)
Comment (BioPhoton) [reply libs/template/src/lib/core/utils/make-zone-less.ts]: done

**Pull request merged**